### PR TITLE
.env migration

### PR DESCRIPTION
**Pull request by Niputi**

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,40 +1,41 @@
 import * as redis from "redis";
 import {CronJob} from "cron";
 import * as rabbitmq from "amqp-ts";
-import {Config, LoadConfigAsync} from "./config"
 import { promisify } from "util";
+import { config } from "dotenv";
+config({
+    path: "../.env",
```
Comment (velddev) [inline src/app.ts]: Will this also work in prod?

File: package.json [hunk 2]
```diff
@@ -4,14 +4,16 @@
   "description": "Shard watchdog for Miki's gateway",
   "main": "src/app.js",
   "dependencies": {
-    "amqp-ts": "^1.4.0",
+    "@types/dotenv": "^6.1.1",
```
Comment (velddev) [inline package.json]: isn't @types/dotenv a devDependency?

**Review by velddev — CHANGES_REQUESTED** [review]
Prod is a built artifact, so it can't go back to the root folder for an env file

**Comment by Niputi** [comment]
I should maybe also add docker in this pr

**Review by velddev — APPROVED** [review]

**Comment by velddev** [comment]
Would recommend doing it in a seperate PR

**Pull request merged**